scripts/brp-python-bytecompile

```diff
@@ -1,6 +1,15 @@
 #!/bin/bash
 errors_terminate=$2
+
+# Usage of %_python_bytecompile_extra is not allowed anymore
+# See: https://fedoraproject.org/wiki/Changes/No_more_automagic_Python_bytecompilation_phase_3
+# Therefore $1 ($default_python) is not needed and is invoked with "" by default.
+# $default_python stays in the arguments for backward compatibility and $extra for the following check:
 extra=$3
+if [ 0$extra -eq 1 ]; then
+    echo -e "%_python_bytecompile_extra is discontinued, use %py_byte_compile instead.\nSee: https://fedoraproject.org/wiki/Changes/No_more_automagic_Python_bytecompilation_phase_3" >/dev/stderr
```

Comment:
Remove the Change reference and explain properly here.

Reply:
Should we drop 8d2f0361ccd0f18f92e7bd71df673b53a229a724 entirely?

Reply:
I'd keep it, but explain it in the error message maybe? Not sure.